devel/util-linux/Portfile

```diff
@@ -80,6 +80,13 @@ configure.args      --disable-agetty \
                     --without-audit \
                     --without-python
 
+if {${configure.build_arch} in [list arm i386 ppc]} {
```

Comment:
Does MacOS run on any 32 bit ARM architecture?

Reply:
Linux and FreeBSD does, I believe (and MacPorts should work on those, at least in principle).
NetBSD does run on arm, but I have no idea if anyone every tried using MacPorts on it.

Reply:
It might work on them in principle but no one is actually doing that and we have no CI to test it.

Reply:
Which is a shame, to be honest, since setting up a Linux or NetBSD CI should be fairly trivial for someone who knows how to set up CI.

I usually do not bother with adding in `arm` into relatively obscure or very complicated ports, but I guess it is sensible to have it in basic ports just in case. This not something I plan to try personally though; of exotic archs I might try RISC-V, otherwise all time goes into PowerPC support.